language/deepseek-r1/docker/Dockerfile.sglang

```diff
@@ -1,41 +1,17 @@
 # SGLang Backend Dockerfile
-FROM nvidia/cuda:12.6.0-devel-ubuntu22.04
+FROM lmsysorg/sglang:v0.5.2-cu129-b200
```

Comment:
Just to confirm - we are using 0.5.2 base container but overwrite with sglang 0.5.4 - is this intended not to use the 0.5.4 container